### ISSUE TITLE
not a problem but a gamemode idea

### ISSUE THREAD

**Issue by digitalhaze** [Sh0wdown/UnofficialCrusaderPatch#386]
Hi Sh0wdown,

as far as u are doing the patches is thought it would be a good idea to tell my idea someone who got the knowledge to maybe has the possibility to implement it.

i also think that it is not soo time intense because there are no new textures needed only some new mechanics.

So here is my idea:

how would it be if the game doesnt end after a mission and instead the opponents disappear after u destroyed them ...the game continues and new (more difficult) opponents spawn (would be nice if u can choose them at that point).

Best regards,
digitalhaze

**Comment by Krarilotus** [comment]
i dont think this will be worked on any time soon, this is definitly alot of work!
